docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -14,6 +14,80 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
+
+Option A
+
+```bash
+# Does not persist across reboots.
+sudo ip rule add from all fwmark 0x200/0xf00 lookup 2004 pref 9  
+sudo ip rule add from all fwmark 0xa00/0xf00 lookup 2005 pref 10  
+sudo ip rule add from all lookup local pref 100  
+```
+
+Option B
+
+```bash
+# Does persist across reboots.
+# Add these values /etc/systemd/networkd.conf
+[Network]
+ManageForeignRoutes=no
+ManageForeignRoutingPolicyRules=no
+```
+
+### Added
+- Add support for AWS CodeCommit repositories in FluxConfig with git configuration [#4290](https://github.com/aws/eks-anywhere/issues/4290)
+- Add new information to the EKS Anywhere Cluster status [#5628](https://github.com/aws/eks-anywhere/issues/5628):
+  - Add the `ControlPlaneInitialized`, `ControlPlaneReady`, `DefaultCNIConfigured`, `WorkersReady`, and `Ready` conditions.
+  - Add the `observedGeneration` field.
+  - Add the `failureReason` field.
+- Add support for different machine templates for control plane, etcd, and worker node in vSphere provider [#4255](https://github.com/aws/eks-anywhere/issues/4255)
+- Add support for different machine templates for control plane and worker node in Nutanix provider [#6291](https://github.com/aws/eks-anywhere/pull/6291)
+- Add support for Kubernetes version 1.25, 1.26, and 1.27 to CloudStack provider [#6167](https://github.com/aws/eks-anywhere/pull/6167)
+- Add bootstrap cluster backup in the event of cluster upgrade error [#6086](https://github.com/aws/eks-anywhere/pull/6086)
+- Add support for organizing virtual machines into categories with the Nutanix provider [#6014](https://github.com/aws/eks-anywhere/pull/6014)
+- Add support for configuring `egressMasqueradeInterfaces` option in Cilium CNI via EKS Anywhere cluster spec [#6018](https://github.com/aws/eks-anywhere/pull/6018)
+- Add support for a flag for create and upgrade cluster to skip the validation `--skip-validations=vsphere-user-privilege`
+- Add support for upgrading control plane nodes separately from worker nodes for vSphere, Nutanix, Snow, and Cloudstack providers [#6180](https://github.com/aws/eks-anywhere/pull/6180)
+- Add preflight validation to prevent skip eks-a minor version upgrades [#5688](https://github.com/aws/eks-anywhere/issues/5688)
+
+### Upgraded
+- Cluster API Provider vSphere: `v1.6.1` to `v1.7.0`
+- Cluster API Provider Cloudstack: `v0.4.9-rc5` to `v0.4.9-rc6`
+- Cluster API Provider Nutanix: `v1.2.1` to `v1.2.3`
+- Cilium: `v1.11.15` to `v1.12.11`
+  NOTE: If you are using the vSphere provider with the Redhat OS family, there is a known issue with VMWare and the new Cilium version that only affects our Redhat variants. To prevent this from affecting your upgrade from EKS Anywhere v0.16 to v0.17, we are adding a temporary daemonset to disable UDP offloading on the nodes before upgrading Cilium. After your cluster is upgraded, the daemonset will be deleted. This note is strictly informational as this change requires no additional effort from the user.
```

Comment:
```suggestion
**Note:** If you are using the vSphere provider with the Redhat OS family, there is a known issue with VMWare and the new Cilium version that only affects our Redhat variants. To prevent this from affecting your upgrade from EKS Anywhere v0.16 to v0.17, we are adding a temporary daemonset to disable UDP offloading on the nodes before upgrading Cilium. After your cluster is upgraded, the daemonset will be deleted. This note is strictly informational as this change requires no additional effort from the user.
```